codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -391,4 +378,38 @@ void writeWindowUpdate(ChannelHandlerContext ctx) throws Http2Exception {
             ctx.flush();
         }
     }
+
+    /**
+     * Provides a means to iterate over all active streams and increment the flow control windows.
+     */
+    private static final class WindowUpdateVisitor implements ActiveStreamVisitor {
+        private CompositeStreamException compositeException;
+        private final int delta;
+
+        public WindowUpdateVisitor(int delta) {
+            this.delta = delta;
+        }
+
+        @Override
+        public boolean visit(Http2Stream stream) throws Http2Exception {
+            try {
+                // Increment flow control window first so state will be consistent if overflow is detected.
+                FlowState state = state(stream);
+                state.incrementFlowControlWindows(delta);
+                state.incrementInitialStreamWindow(delta);
+            } catch (StreamException e) {
+                if (compositeException == null) {
+                    compositeException = new CompositeStreamException(e.error(), DEFAULT_COMPOSITE_EXCEPTION_SIZE);
```

Comment:
@Scottmitch I know it's been here before but do we really need the `DEFAULT_COMPOSITE_EXCEPTION_SIZE`?